odxtools/utils.py

```diff
@@ -34,8 +36,7 @@ def short_name_as_id(obj: Any) -> str:
     prepends an underscore to such such shortnames.
     """
 
-    sn = obj.short_name
-    assert isinstance(sn, str)
+    sn: str = odxrequire(obj.short_name)
```

Comment:
checking something is not None and something is string is not the same thing

Reply:
good point. fixed.